src/settings/TargetProfiles.test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+
+import '../../test/jest/__mock__';
+
+import {
+  renderWithIntl,
+  stripesStub,
+  translationsProperties
+} from '../../test/jest/helpers';
+
+import TargetProfiles from './TargetProfiles';
+
+jest.unmock('@folio/stripes/components');
+
+
+const resources = {
+  entries: {
+    type: 'okapi',
+    records: [{
+      displayName: '✓test',
+      enabled:true,
+      name:'profiles'
+    }, {
+      displayName: '✕test',
+      enabled:true,
+      name:'profiles'
+    }],
+    path: 'copycat/profiles',
+    resourceShouldRefresh: true,
+    GET: {
+      path: 'copycat/profiles?query=cql.allRecords=1&limit=1000',
+    },
+  }
+};
+
+const mutator = {
+  entries: {
+    PUT: jest.fn(),
+    POST: jest.fn(),
+    DELETE: jest.fn(),
+  }
+};
+
+const defaultProps = {
+  stripes: {
+    ...stripesStub,
+    connect: component => component,
+  },
+  label:'test label',
+  resources,
+  mutator,
+};
+const TargetProfilesSetup = () => (
+  <MemoryRouter>
+    <TargetProfiles {...defaultProps} />
+  </MemoryRouter>
+);
+const renderTargetProfilesSettings = () => renderWithIntl(
+  <TargetProfilesSetup />,
+  translationsProperties
+);
+describe('TargetProfiles Component', () => {
+  it('should take snapshot', () => {
+    const container = renderTargetProfilesSettings();
+    expect(container).toMatchSnapshot();
```

Comment:
I'm not familiar with `toMatchSnapshot()`. The [snapshot documentation](https://jestjs.io/docs/snapshot-testing#snapshot-testing-with-jest) I found notes, 
> The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. 

It looks like that didn't happen here. If you do go this route, please explain it in the PR where you introduce this pattern for the first time since it's a new test pattern. Please also explain why existing patterns are insufficient and/or why this one is better. You'll also need to investigate how we can exclude the `__snapshots__` directories from the artifacts we publish to NPM. To be clear, it is absolutely fine to introduce this new pattern, but you need to (a) do it correctly and (b) explain why you're doing it. 

Alternatively, see if there are other `EntryManager` tests. You might find it is sufficient to mock it as a simple component that echoes a few props (e.g. `label`) and your test can simply render and then check
```
expect(someProp).toBeInTheDocument();
```